filter/src/main/resources/reference.conf

```diff
@@ -0,0 +1,89 @@
+// Reference configuration of metric filters and predefined default values.
+//
+// - Metrics are organized to a tree structure that may split on any level.
+// - Root node is virtual and defines defaults that may be later overwritten on any level.
+// - Branch nodes may be used to configure subtree of metrics, they may also hold metrics.
+// - Leaf nodes are metrics like counters and timers that carry values.
+// - Each metric may compute and provide fields like count, rate, mean, percentiles, etc.
+// - Configurations from all levels are merged together, the most specific one wins.
+// - Separator for node names in this configuration is '/', separator used in real is implementation specific.
```

Comment:
- MetricsMonitor already uses "/". I wanted to be compatible with the rest of the code.
- I initially wanted "." too (Graphite style), but it would probably conflict with Typesafe config.
- Prometheus uses "_" as a separator, but it's more for separating of words than parts of names.